### DISCUSSION TITLE
Add Config option for logging

### DISCUSSION THREAD

**Discussion by MisterMountain** [nginx-proxy/nginx-proxy#2328] [Ideas]
Hi!

I think it would be nice to have a config option to enable logging, especially the accesslog, into /var/log/nginx/accesslog, or to a volume on the host - or is it already possible?

**Comment by SchoNie** [comment]
acceslog and error logs are written by default. Just map them to a volume or bind mount to save them on the host.
See [Log to a file #2190](https://github.com/nginx-proxy/nginx-proxy/discussions/2190) for example or do a search to find more examples.

If you need a custom log format. You can [set LOG_FORMAT=xxx on the proxy container.](https://github.com/nginx-proxy/nginx-proxy#custom-log-format)
